.github/workflows/build-backend.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Push Backstage Docker Image
+
+on:
+  repository_dispatch:
+    types: [backstage_release]
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install Dependencies
+        run: yarn install --immutable
+
+      - name: Build Backend
+        run: |
+          yarn tsc:full
+          yarn build:all
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: bot-env0
```

Comment:
Most examples online use the dispatch actor name (it'd be bot-env0 tho)

Reply:
👍 
https://github.com/env0/env0-backstage-plugin/pull/81/commits/4a24cf086dc33d52540cc68a99f86f1204dde96a